webhooks/virtualmachine/v1alpha1/validation/virtualmachine_validator_unit_test.go

```diff
@@ -422,99 +427,100 @@ func unitTestsValidateCreate() {
 	annotationPath := field.NewPath("metadata", "annotations")
 
 	DescribeTable("create table", validateCreate,
```

Comment:
From the v1a2 work there is a pending item to start to move away from the table driven tests here because they have become IMO untenable and unreadable for a lot of stuff (there are a lot more network and bootstrap stuff in v1a2 that I have no desire to express with bools).

How about for this MR we leave the table stuff as-is (and add an assert in validateCreate that warning is always empty) to keep this diff smaller & focused, and lay down the start of individual Context/It?